docs/topics/practices.rst

```diff
@@ -119,7 +119,6 @@ Here is an example that runs multiple spiders simultaneously:
 
     import scrapy
     from scrapy.crawler import CrawlerProcess
-    from scrapy.utils.project import get_project_settings
```

Comment:
Changes in this file seem unrelated to the other changes in the PR about import paths. Are they necessary? Could they be taken out to a different PR?

Reply:
These changes were dragged in this PR because of [427bf73](https://github.com/scrapy/scrapy/pull/5099/commits/427bf73850ad79a36a2fe7c549246f2d9f57be2f), which is one of the commits that had occurred due to a rebase. 

Indeed, they are out of this PR scope. Could we remove these lines? I guess they already belong to another PR (as they are included in the [practices.rst](https://github.com/scrapy/scrapy/blob/master/docs/topics/practices.rst) file of scrapy master branch).

Reply:
Hello @elacuesta.These changes are out of the  PR scope and as @marlenachatzigrigoriou is pointing out, happend due to the above two commites (427bf73 and e66fbc4) .
We have now fixed it and in "files changed" there are no changes in the practices file.

Reply:
Thanks for the update!